src/services/assetService.ts

```diff
@@ -48,7 +91,20 @@ export class AssetService {
 
         // eslint-disable-next-line
         const extensionParts = fileNameParts[fileNameParts.length - 1].split(/[\?#]/);
-        const assetFormat = extensionParts[0];
+        let assetFormat = extensionParts[0];
+
+        if (supportedImageFormats.hasOwnProperty(assetFormat)) {
+            const mime = await this.getMimeType(filePath);
+
+            // If file was renamed/spoofed - fix file extension to true MIME type and show message
+            if (assetFormat !== mime) {
+                assetFormat = mime;
+                const corruptFileName = fileName.split("%").pop().substring(2);
```

Comment:
Doesn't account for file names with spaces 
![image](https://user-images.githubusercontent.com/60453211/81883094-36a29f80-9549-11ea-8fb4-c5716a4a4d38.png)